### DISCUSSION TITLE
Video wallpapers into still photos as seen in macOS Sonoma

### DISCUSSION THREAD

**Discussion by joshuafhiggins** [t1m0thyj/WinDynamicDesktop#543] [Ideas]
During WWDC, a new feature was shown for macOS Sonoma where, on the lock screen and desktop, a slow motion, 5K video would play of the desktop wallpaper as it transitions to a still image once reaching your desktop. While I know the project has no plans to support video wallpapers (#26), I was wondering if this native macOS feature would change things. I am also able to work on the feature and submit a pull request, if you would like.

**Comment by t1m0thyj** [comment]
This is a cool feature but seems significantly different from the idea of dynamic desktop wallpapers. Because of this I would be hesitant to integrate this feature into WinDD. I'm also unsure how many video wallpapers are readily available, and how to display them on the Windows lockscreen since the available APIs only support still images AFAIK. All that being said, feel free to submit a pull request with a PoC if you have an idea of how to implement the feature and are interested 🙂
